src/network-uptime/network-dashboard.py

```diff
@@ -6,32 +6,52 @@
 import matplotlib.pyplot as plt
 from matplotlib.ticker import MaxNLocator
 import requests
+from datetime import datetime
 import streamlit as st
 
 dotenv_path = os.path.join(os.path.dirname(__file__), '.env')
 load_dotenv(dotenv_path)
 
 BASE_API_URL = os.getenv('BASE_URL')
-START_QUERY_DATE = os.getenv('START_DATE')
-END_QUERY_DATE = os.getenv('END_DATE')
 
 matplotlib.use('Agg')
 
 st.set_page_config(page_title='NETWORK UPTIME AND DOWNTIME', layout="wide")
-st.title('Data report on network uptime and outages over the first quarter, 2022')
+st.title('Data report on network uptime and outages')
 
 
 api_url = f"{BASE_API_URL}/devices?tenant=airqo"
 results = requests.get(api_url)
 devices_data = results.json()["devices"]
 
+st.sidebar.markdown(
+    "<h1 style='text-align: center;'>SELECT QUERY DATE</h1>", unsafe_allow_html=True)
+side_row1_col1, side_row1_col2 = st.sidebar.columns([1, 1])
+with side_row1_col1:
+    selected_start_date = st.date_input(
+        'start date', datetime(2020, 1, 1))
```

Comment:
Thanks @uman95.
Small suggestion though. Could the default start date be set to the date 3 months ago. i.e datetime.now() - 3 months?

Reply:
Thanks @123MwanjeMike. Done